docs/zfp.rst

```diff
@@ -0,0 +1,17 @@
+Zfp
```

Comment:
Not to bikeshed things, but are we wanting this to be `zfp` or `zfpy`? Ok either way, just wanted to check that we are happy with the naming before we ship this.

Reply:
If fact, the PR #160  is not needed any more, because @lindstro has their own zfp wrapper, so I have PR #229 ready for you to approve to merge. @jakirkham

Reply:
Sorry, must have gotten turned around. Should we close this PR out then?